packages/jspsych/tests/citations/citations.test.ts

```diff
@@ -16,12 +16,16 @@ let JsPsych;
  * This is because the citation functionality is only available in the built version
  * due to code injections that run during the build.
  */
-
 try {
   // Try to import built version
   JsPsych = require("../../dist/index").JsPsych;
   let jspsych: typeof JsPsych;
 
+  // Check if getCitations exists in current built version
+  if (!jspsych.hasOwnProperty("getCitations")) {
+    throw new Error("getCitations not found");
```

Comment:
maybe suggest rebuilding with `npm run build` if jsPsych 8.2+?

Reply:
Jest doesn't rely on the rollup build but transpiles the source files itself based on its module name mapper, according to the aliases in our TS config. I don't see why this check would fail :thinking:

Reply:
For this one test suite we made it use the built version (L21) because we inject the citation info from the citation.cff file.

Reply:
Sorry, my bad! I'd like to voice my concerns about implicitly testing the build setup in unit tests though :melting_face: If I'm not mistaken, this test would fail as soon as the main citation is changed. If I may voice a wish, I'd much prefer to test citation building in the config package, and make this test run against mocked citation strings :upside_down_face: